packages/react/src/components/Dialog/Dialog-story.js

```diff
@@ -7,149 +7,75 @@
 
 /* eslint-disable storybook/story-exports */
 
-import * as React from 'react';
-import { FocusScope } from '../FocusScope';
-import { Dialog } from '../Dialog';
-import { useId } from '../../internal/useId';
-import { Portal } from '../Portal';
+import React, { useEffect, useState } from 'react';
+import Dialog from './';
+import Button from '../Button';
+import { action } from '@storybook/addon-actions';
 
 export default {
   title: 'Experimental/unstable_Dialog',
   component: Dialog,
-  includeStories: [],
```

Comment:
Adding this back in will ensure these stories don't show up in the storybook for now